monitor/grafana/zeebe.json

```diff
@@ -3615,6 +3617,201 @@
       },
       "id": 42,
       "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_PROMETHEUS}",
+          "description": "Shows the GC count per second, aggregated per pod and GC type.",
```

Comment:
:question: I don't know if the description helps much. What is GC count per second, and how should I interpret it? Is it literally how often it runs per second?

Reply:
Yes it is `literally how often it runs per second` :D Any suggestion how to improve the message?

Reply:
Maybe instead of GC count, we take what the docs say and rephrase as `the number of collections per second`? Or the `rate of collections`? It's just GC count part that's clear to me, I think `collection count` makes more sense or some such.

Reply:
Ok I can try to change it. BTW GC means garbage collection, which would mean `GC count = garbage collection count`. I think this is similar to your last example.

Reply:
Is GC garbage collection or garbage collector? :exploding_head:

Reply:
Depends on the context :)

Reply:
https://en.wikipedia.org/wiki/Garbage_collection_(computer_science) GC = garbage collection - isn't wikipedia the truth :laughing:

Reply:
Hm, but then you have https://en.wikipedia.org/wiki/Boehm_garbage_collector or as Wikipedia calls it, Boehm GC :wink: